.claude/hooks/utils/llm/ollama_provider.py

```
@@ -0,0 +1,193 @@
+        env_path = os.path.join(check_dir, ".env")
+        possible_env_paths.append(env_path)
+        check_dir = os.path.dirname(check_dir)
+        if check_dir == "/":  # Reached root
+            break
+    
+    # Try loading from each possible location
+    env_loaded = False
+    for env_path in possible_env_paths:
+        if os.path.exists(env_path):
+            load_dotenv(env_path)
+            env_loaded = True
```

Ruff (0.12.2)

Local variable `env_loaded` is assigned to but never used

Remove assignment to unused variable `env_loaded`

(F841)